certora/specs/SummaryLib.spec

```diff
@@ -4,20 +4,19 @@ using MorphoHarness as MORPHO;
 using Util as Util;
 
 methods {
-    function MORPHO.market_(PreLiquidation.Id) external
-        returns (PreLiquidation.Market memory) envfree;
-    function MORPHO.position_(PreLiquidation.Id, address) external
-        returns (PreLiquidation.Position memory) envfree;
+    function MORPHO.virtualTotalBorrowAssets(PreLiquidation.Id) external returns (uint256) envfree;
+    function MORPHO.virtualTotalBorrowShares(PreLiquidation.Id) external returns (uint256) envfree;
+    function MORPHO.borrowShares(PreLiquidation.Id, address) external returns (uint256) envfree;
+    function MORPHO.collateral(PreLiquidation.Id, address) external returns (uint256) envfree;
```

Comment:
Just curiosity, why do you like this style better rather than accessing values through the structs?

Reply:
I find that it's less overhead when things are simple. If you plan to use only one variable in that struct, which was mostly the case for these getters, then you don't need to give another name to the struct you are getting (which was `m` or `p` a lot of the time). See for example the `MarketExists.spec` and how it's more concise now

Reply:
Thanks for the answer :D ! I agree that often concise code is more pleasurable to read, but I'm not on the side of "concise is always more simple". Sometimes (not the case here) concise code be tremendously complicated to understand in contrast with  fully explicit code. For example here you have a concise [program](https://github.com/ioccc-src/winner/blob/master/2000/tomx.c) that is, at the same time, a C source file, a Makefile and a Shell script to compile and execute itself.